src/ChipInput.js

```diff
@@ -635,13 +715,25 @@ ChipInput.defaultProps = {
 
 export default withStyles(styles)(ChipInput)
 
-export const defaultChipRenderer = ({ value, text, isFocused, isDisabled, handleClick, handleDelete, className }, key) => (
+export const defaultChipRenderer = ({ value, text, isFocused, isDisabled, handleClick, handleDelete, className, color }, key) => (
   <Chip
     key={key}
     className={className}
     style={{ pointerEvents: isDisabled ? 'none' : undefined, backgroundColor: isFocused ? blue[300] : undefined }}
     onClick={handleClick}
     onDelete={handleDelete}
     label={text}
+    color={color || 'default'}
```

Comment:
The default value for `color` could be specified in the component's `defaultProps` so that it appears in the documentation, too.